cpmpy/transformations/linearize.py

```diff
@@ -445,6 +435,66 @@ def only_positive_bv(lst_of_expr):
 
     return newlist
 
+def only_positive_bv_sub(expr):
```

Comment:
I would change the name of this helper function, now the name can be confused as "it only works for subtraction"

Maybe smt like `only_postive_bv_num`, or `only_positive_bv_linear`